data-processing/common/parse_equation.py

```diff
@@ -53,9 +57,12 @@ def get_symbols(mathml: str) -> List[Symbol]:
     node_symbols = []
     consecutive_char_sequence: List[BeautifulSoup] = [] # The symbol_nodes of consecutive characters
     consecutive_char_indices: List[int] = [] # Indices of the characters of a consecutive character sequence
-    # It's 'safe' to call soup.find_all as the underlying bs4 implementation 
+    # It's 'safe' to call soup.find_all as the underlying bs4 implementation
```

Comment:
Hahaha I'm just noticing the foreboding quotes around 'safe' here XD

Reply:
XD